qemu/tests/sgx_basic.py

```diff
@@ -0,0 +1,42 @@
+from virttest import error_context
+from provider.sgx import SGXHostCapability
+from virttest.utils_misc import verify_dmesg
+from provider.sgx import SGXChecker
```

Comment:
from virttest import error_context
from virttest.utils_misc import verify_dmesg

from provider.sgx import SGXChecker
from provider.sgx import SGXHostCapability

Reply:
Thanks, I will change it